main.go

```diff
@@ -111,8 +112,8 @@ Usage:
   ios apps [--system] [--all] [--list] [--filesharing] [options]
   ios launch <bundleID> [--wait] [options]
   ios kill (<bundleID> | --pid=<processID> | --process=<processName>) [options]
-  ios runtest [--bundle-id=<bundleid>] [--test-runner-bundle-id=<testrunnerbundleid>] [--xctest-config=<xctestconfig>] [--env=<e>]... [options]
-  ios runwda [--bundleid=<bundleid>] [--testrunnerbundleid=<testbundleid>] [--xctestconfig=<xctestconfig>] [--arg=<a>]... [--env=<e>]... [options]
+  ios runtest [--bundle-id=<bundleid>] [--test-runner-bundle-id=<testrunnerbundleid>] [--xctest-config=<xctestconfig>] [--raw-testlog] [--env=<e>]... [options]
```

Comment:
`--raw-testlog` isn't really self explanatory and it's not explained anywhere. Maybe something like `--log-output=<file>` makes more sense and document that passing `-` prints them to `stdout`

Reply:
Removed it and added `--log-output` with docs.